command/results_xunit_parser_test.go

```diff
@@ -274,16 +296,13 @@ func TestXMLToModelConversion(t *testing.T) {
 				So(passCount+skipCount+failCount, ShouldEqual, len(tests))
 
 				Convey("and logs should be of the proper form", func() {
-					grip.Info(logs[0])
-					grip.Info(logs[3])
+					grip.Info(logs)
 					So(logs[0].Name, ShouldEqual, "unittest.loader.ModuleImportFailure.tests.test_binder")
 					So(len(logs[0].Lines), ShouldNotEqual, 0)
-					Convey("and system-out and system-err should be present", func() {
-						So(logs[len(logs)-1].Lines[0], ShouldContainSubstring, "system-err:")
-						So(logs[len(logs)-2].Lines[0], ShouldContainSubstring, "system-out:")
-					})
 				})
 			})
 		})
 	})
 }
+
+//TODO: implement a mock logger and test the sendLogsForOneFile function
```

Comment:
let's do this, and also maybe try to do some sort of higher level integration test case? and or write a test in the agent that calls into this code in the agent/agent_command_test.go file?